README.md

```diff
@@ -1,95 +1,116 @@
-# jpo-sdw-depositor [![Build Status](https://travis-ci.org/usdot-jpo-ode/jpo-sdw-depositor.svg?branch=dev)](https://travis-ci.org/usdot-jpo-ode/jpo-sdw-depositor) [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=usdot.jpo.ode%3Ajpo-sdw-depositor&metric=alert_status)](https://sonarcloud.io/dashboard?id=usdot.jpo.ode%3Ajpo-sdw-depositor)
+# jpo-sdw-depositor
 
-Subscribes to a Kafka topic and deposits messages to the Situation Data Warehouse (SDW).
+Subscribes to a Kafka topic and deposits messages to the [Situational Data Exchange (SDX)](https://sdx.trihydro.com/).
```

Comment:
Not sure I understand what is the difference between the SDW and SDX. Do we need to rename this jpo-sdw-depositor to jpo-sdx-depositor?

Reply:
The SDX used to be called the SDW, but has since been renamed. Ideally the repository would be renamed to `jpo-sdx-depositor` to match, yeah.